TankDriver.App/Models/BulletSpaceModel.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using TankDriver.Logic;
+
+namespace TankDriver.Models
+{
+	internal class BulletSpaceModel: IModel
+	{
+		private BulletSpace _bulletSpace;
+		private TextureStorage _textureStorage;
+
+		public BulletSpaceModel (BulletSpace bulletSpace)
+		{
+			_bulletSpace = bulletSpace;
+		}
+
+		public void LoadBulletTextures(IModel bulletModel)
+		{
+			bulletModel.LoadTextures (_textureStorage);
+		}
+
+		#region IModel implementation
+
+		public IUnit GetUnit ()
+		{
+			return _bulletSpace;
+		}
+
+		public void LoadTextures (TextureStorage textureStorage)
+		{
+			_textureStorage = textureStorage;
+		}
+
+		public void Render (SpriteBatch spriteBatch)
+		{
+			foreach (var bullet in _bulletSpace.Bullets) {
+				bullet.GetModel ().Render (spriteBatch);
+			}
+		}
+
+		#endregion
+	}
+}
+
```

Comment:
Do we really need this `BulletSpaceModel`? I think my initial idea was to have a model for every unit (e.g. tank, aircraft, bullet or missile) and not for abstract entities like that.

Yes, we do have need to store a list of models somewhere, but let's maybe just store the list in the `Game` class itself? And periodically call `Render` and `Update` for each model on the list (including the `TankModel` itself).